crates/wasm-smith/src/config.rs

```diff
@@ -232,6 +232,12 @@ pub trait Config: for<'a> Arbitrary<'a> + Default + Clone {
         false
     }
 
+    /// Determines whether the SIMD proposal is enabled for
+    /// generating insructions. Defaults to `true`.
+    fn simd_enabled(&self) -> bool {
+        true
```

Comment:
I think this may want to be false for now while it isn't otherwise enabled by default for other tools? One of the main consumers, wasmtime, also hasn't yet finished the implementation so we would need to explicitly turn this off.

Reply:
Sure, changed to `false` in 15fd226. I was assuming it was going to be default-enabled soon enough but either way is fine. But I would like to start trying to find SIMD fuzz bugs in Wasmtime sooner rather than later: is there any way to turn it on and then ignore `unimplemented!` errors?